src/web/event-notifications/event-notification-types/SlackNotificationForm.jsx

```diff
@@ -77,6 +79,7 @@ class SlackNotificationForm extends React.Component {
       link_names: false,
       icon_url: '',
       icon_emoji: '',
+      backlog_size: 5,
```

Comment:
In the backend the default `backlogSize` is 0. I think we should use the same in the backend and frontend.

Reply:
nick gave a suggestion to default the value to `5' I have changed the backend to default it to 5 as well.

Reply:
its reverted back to 0  in this [commit]( https://github.com/Graylog2/graylog-plugin-integrations/pull/639/commits/2f2efcfef97a5b3d7ce78a0866a93148bb4eb5ff)